app/components/tooltip.tsx

```diff
@@ -7,6 +7,7 @@ import { Explainer } from './icons/explainer';
 export const ExplainerTooltip: FC<BoxProps> = ({ children, ...props }) => {
   return (
     <Tippy
+      zIndex={9999999}
```

Comment:
![nice](https://user-images.githubusercontent.com/11803153/105741148-98504180-5eff-11eb-8b35-01619665bae7.gif)

Reply:
Needs a few more 9's just to be sure.

Reply:
This was a lazy fix.